src/browser/components/SkillItem.react.js

```diff
@@ -31,15 +31,29 @@ export default class SkillItem extends Component {
     const STARS_LENGTH = 5;
     for (let i = STARS_LENGTH; i >= 1; i--) {
       starsLabels.push(
-          <input className={`star star-${i}`} id={`star${i}_${this.props.data.id}`} key={i} name="star" onChange={(e) => this.handleStarChange(i, e)} type="radio"/>,
-          <label className={`star star-${i}`} htmlFor={`star${i}_${this.props.data.id}`} key={i + STARS_LENGTH} />
+          <input className={`star star-${i}`}
+                 id={`star${i}_${this.props.data.id}`}
+                 key={i}
+                 name="star"
+                 onChange={(e) => this.handleStarChange(i, e)}
```

Comment:
I think we can distinguish stars from `e` alone. Then `this.handleStarChange` will do the trick